### DISCUSSION TITLE
빌드세팅에 등록된 씬 이름이 출력되지 않습니다......ㅠㅠ

### DISCUSSION THREAD

**Discussion by DrunkenPoy** [naverz/zepeto-studio-kor#812] [5. Q&A]
전체 씬이름을 저장하는 배열을 만들고 싶은데요,,,,,

![image](https://user-images.githubusercontent.com/55507599/170642598-a4b366d7-5447-413c-baa1-08068d41f85c.png)

구현된 코드는 이렇습니다. 
![sceneb](https://user-images.githubusercontent.com/55507599/170642041-524b50a7-2652-4eb4-843d-789df35517c1.PNG)
빌드세팅은 이렇게 되어 있구요.
아니 근데....해당코드 유니티에서는 정상적으로 출력됩니다
![scene](https://user-images.githubusercontent.com/55507599/170642290-001dfb63-2737-4605-8ce2-30213fb31a5d.PNG)
ㄴ 이렇게 정상적으로 나옵니다만

![sceneM](https://user-images.githubusercontent.com/55507599/170642326-f86f3109-9497-44e0-82ee-93238f1db807.png)
ㄴ 모바일에서는 이렇게 나옵니다.

의도된 동작인지 버그인지는 모르겠습니다만 빌드세팅에 있는 씬 이름과 인덱스를 가져오는 다른 방법이 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 작업하신 씬들은 에셋번들 형태로 빌드되어 재패토앱에서 실행되기 때문에 현재로써는 다르게 동작하게 됩니다.
다만 제보해준 내용으로 문제를 인지했고 의도하신 기능으로 동작할 수 있도록 검토하겠습니다.
감사합니다.
